### DISCUSSION TITLE
欢迎反馈

### DISCUSSION THREAD

**Discussion by KouShenhai** [KouShenhai/KCloud-Platform-IoT#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 1czy** [comment]
maven版本用的多少啊，为什么我编译laokou-common包一直报错“无法访问org.springframework.context.support.ReloadableResourceBundleMessageSource”；我都把依赖重新删除导入好多次了，这应该不用阿里云私服，和nexus把；

**Reply by KouShenhai** [reply]
maven 3.9.0

**Reply by KouShenhai** [reply]
> maven版本用的多少啊，为什么我编译laokou-common包一直报错“无法访问org.springframework.context.support.ReloadableResourceBundleMessageSource”；我都把依赖重新删除导入好多次了，这应该不用阿里云私服，和nexus把；

报错麻烦给一下截图，我这边好定位问题
